docs/user-guide/members/README.md

```diff
@@ -8,16 +8,62 @@ keywords: members, project participants
+
+![Project members overview in OpenProject](members-overview.png)
+
+Standard filters on the left side menu include the following: 
+
+- **All** - returns all members and groups of the project, as well as non-members, with whom a work package from this project has been shared
+
+- **Locked** - returns all locked users that are members of this project, as well as locked non-members, with whom a work package from this project has been shared
+
+- **Invited** - returns all users that have been invited, but have not yet registered
+
+- **Project roles** provides filters based on all the user roles that have been assigned to users in that specific project. 
+
+- **Work package shares** provides filters based on all the roles available for sharing work packages. They include
+- **All shares** - returns all users that a work package in this project has been shared with
+  
+- **View** - returns all users that can view, but not edit or comment on a work package that has been shared with them
+  
+- **Comment** - returns all users that are allowed to add comments to a work package that has been shared with them
+  
+- **Edit** - returns all users that are permitted to edit a work package that has been shared with them
+
+> Note: users, with whom work packages from a given project have been shared, can not be edited or deleted under **Members**. To edit or revoke their viewing rights please navigate to **Work packages**, select the **Shared with users** filter and adjust the privileges accordingly. [Read more](../work-packages/share-work-packages/#remove-sharing-privileges).
+
+- **Groups** lists all the groups that have been added to this project. 
+
+> Note: members that are part of a group will also be displayed as members individually. In that case you can only edit the roles assigned to the users, but not delete him or her. If you want to delete a user that is a member of a group (also added to this project) you will have to delete the entire group and add group members individually if needed.
+
+![Group project member in OpenProject](members-overview-group.png)
+
+You can adjust the overview by clicking the **menu icon** in the top right corner next to **+ Member** button. Once you are done adjusting your preferences, click the blue **Apply** button.
+
+![Filter project members overview](filter-project-members.png)
+
+You can adjust the project member overview based on following filters:
+
+- **Status** - allows filtering based on the user status, such as active, invited, locked or registered.
+- **Group** - allows filtering for project members that are part of an existing group (all groups available in your instance will be listed as options), even if the group itself has not been added to the project.
+- **Role** - allows filtering based on all the user roles that have been assigned to users in that specific project. This will duplicate the options in the left side menu.
```

Comment:
I would change this sentence:

> This will duplicate the options in the left side menu.

to something like "The options of this filters are the same as in the left side menu."